### ISSUE TITLE
bug: fast-tree-item does not indicate affirmative selected state using screen reader

### ISSUE THREAD

**Issue by stevenbenack** [microsoft/fast#4127]
**Describe the bug; what happened?**
Using NVDA, tabbing through a fast-tree-view for tree-items, screen reader announces aria-selected=false, but does not announce aria-selected=true.

**What are the steps to reproduce the issue?**
1. Boot NVDA screen reader
2. Go to FAST component explorer, https://explore.fast.design/components/fast-tree-view
3. Select a tree-item
4. Tab into the selected tree-item
5. See that NVDA doesn't announce that the tree-item is selected even though `aria-selected=true`
6. Tab to an unselected tree-item where `aria-selected=false`
7. See that NVDA explicitly announces that the item is unselected

**What behavior did you expect?**
Screen reader should indicate that it is currently selected.

**In what environment did you see the issue?**
Repro in Firefox 78, Edge Anaheim 86, Chrome 86

**Comment by chrisdholt** [comment]
Thanks @stevenbenack for the issue. To be honest, I think we can investigate, but I'm not really sure what to make of this. From our standpoint, the component has the correct role and includes the correct attributes with valid states. We're providing a valid UIA representation as far as I can tell. Does this repro in other AT (JAWS, Narrator) or just NVDA? Note, VoiceOver calls out "listbox" for tree view, so no need to check VoiceOver as it has its own problems.

**Comment by chrisdholt** [comment]
@stevenbenack after a quick search, I think I've found this to be an issue with NVDA: https://github.com/nvaccess/nvda/issues/8587. I think this is external to FAST so I'm going to close as such.

Thanks!

**Comment by chrisdholt** [comment]
Closed as not-actionable by FAST - External to AT.
